docusaurus/docs/Android/02-client/06-guides/06-offline-support.mdx

```diff
@@ -0,0 +1,76 @@
+# Offline Support
+
+The offline library provides a caching mechanism applied automatically to some of the `ChatClient` functions as side effects.
+The offline results, as well as online results, are being exposed through `StateFlow` objects that can be accessed using state-related objects.
```

Comment:
```suggestion
The offline results, as well as online results, are being exposed through `StateFlow` objects that can be observed for changes.
```

Collected is the technically correct terminology, but the observer pattern is not limited to any implementation so it's correct aswell and more recognized too.